bluechi.spec.in

```diff
@@ -167,27 +173,21 @@ SELinux policy associated with the bluechi and bluechi-agent daemons
 %post selinux
 # Remove hirte policy
 if [ $1 -eq 1 ]; then
-   semanage port -N -d -p udp 842 2>/dev/null || true
-   semanage port -N -d -p tcp 842 2>/dev/null || true
```

Comment:
Don't we need to keep selinux upgrade from hirte to bluechi till Fedora 39 deprecation? I thought that we would want to add a build option to be able to turn it off on selected platforms?

Also, have you tried upgrade from bluechi 0.7 to bluechi 0.8 with any port policy update? Don't we need to remove those specific port options before installing new policy with any port?

Reply:
These two lines for semanage basically remove the allow rule for tcp and udp on port 842 - no specific domain is given. I don't think these are needed even for the old hirte considering the latest changes. 

> Also, have you tried upgrade from bluechi 0.7 to bluechi 0.8 with any port policy update? Don't we need to remove those specific port options before installing new policy with any port?

Based on how the any port is implemented:
```
tunable_policy(`bluechi_agent_port_connect_any',`
	corenet_tcp_connect_all_ports(bluechi_agent_t)
',`
	allow bluechi_agent_t bluechi_agent_port_t:tcp_socket name_connect;
')
```
Either the labelled port or the any connect/bind is used. Therefore, even though the port labels aren't removed, those don't have any effect till the any is disabled. And even then, the ports would be "pre-configured" - which is ok, I think, considering the new documentation would guide the user to add ports (and thus viewing these). 

I tried verifying this by upgrading from v0.7 to v0.8 with the any port policy and it really kept the port rules:
```bash
[root@localhost tmp]# semanage port -l | grep bluechi
bluechi_port_t                 tcp      842
bluechi_port_t                 udp      842
```
And starting BlueChi on another port was still possible with the any policy active. Switch it off and running the controller on another port, e.g. 8420, was blocked as expected.

Reply:
OK, makes sense